aiida/cmdline/utils/decorators.py

```diff
@@ -47,10 +47,7 @@ def load_backend_if_not_loaded():
         with spinner():
             load_profile()  # This will load the default profile if no profile has already been loaded
             manager.get_backend()  # This will load the backend of the loaded profile, if not already loaded
-            # Loading the backend will have reconfigured the logging (see ``Manager._load_backend`` which calls the
-            # ``aiida.common.log.configure_logging`` function but with the default ``cli=False``), so here we have to
-            # call it again to ensure that the correct CLI handlers and formatters are configured.
-            configure_logging(with_orm=True, cli=True)
+            configure_logging(with_orm=True)
```

Comment:
if the `get_backend` call already configured the logging, is it still necessary to do it again?

Reply:
Good point, not sure where this got added first, but it seems to be doubling up indeed. Will check to make sure and then remove it if really not needed

Reply:
Checked and removed it